c7n/resources/appflow.py

```diff
@@ -0,0 +1,119 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+from c7n.actions import BaseAction
+from c7n.manager import resources
+from c7n.query import QueryResourceManager, TypeInfo
+from c7n.tags import RemoveTag, Tag, TagActionFilter, TagDelayedAction
+from c7n.utils import local_session, type_schema
+
+
+@resources.register('app-flow')
+class AppFlow(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = 'appflow'
+        arn_type = 'flow'
+        enum_spec = ('list_flows', 'flows', {'maxResults': 100})
+        id = name = 'flowName'
+        arn = 'flowArn'
+        detail_spec = ('describe_flow', 'flowName', 'flowName', None)
+        not_found_err = 'ResourceNotFoundException'
```

Comment:
each class attribute on resource_type is documented in query.py.. afaicr this is not one of those. it could be reasonable to add though.

Reply:
given its only used in one place in this module perhaps better to inline to usage.

Reply:
thanks, i shamefully copied https://github.com/cloud-custodian/cloud-custodian/blob/5f0ce6ec7130806d459d52e9eca0dda37120c29d/c7n/resources/qldb.py#L54 but I've fixed this issue.